transformations/noun_compound_paraphraser/transformation.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+import os
+import json
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
```

Comment:
Please add docstrings, precising the ourpose of the function, the format description of its aruments and outputs.